examples/microprofile/hello-world-explicit/README.md

```diff
@@ -13,5 +13,8 @@ Then try the endpoints:
 
 ```
 curl -X GET http://localhost:7001/helloworld
-curl -X GET http://localhost:7001/helloworld/another
+curl -X GET http://localhost:7001/helloworld/earth
 ```
+
+By default the server will use a dynamic port, see the messages displayed
+when the application starts.
```

Comment:
Haven't looked deeply but down below you set the port to 7001, so will a dynamic port truly be used?

Reply:
In the explicit example it uses a dynamic port. In the implicit example it uses 7001.